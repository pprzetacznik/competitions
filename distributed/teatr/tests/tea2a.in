100000000 0 -1 -1 5 2906159023179899071
