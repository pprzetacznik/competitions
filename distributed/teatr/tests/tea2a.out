2000011295263335
